### PR TITLE
LPS-181590 | Add a testcase to delete API Application Schema

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Add a testcase to delete API Application Schema

Test Plan
CanSeeWarningMessageBeforeSchemaIsDeleted
Given API Application with Schema created
When delete API Schema
Then I can see the Delete API Schema dialog with message

CanCancelAPISchemaDeletion
Given API Application with Schema created
And Given delete API Schema
When I click cancel button
Then I can see the API Schema

CanDeleteAPISchema
Given API Application with Schema created
And Given goto dropdown menu and click delete
When I click delete button
Then the API Schema has been deleted in the UI

CannotSeeSchemaInAPIExplorerAfterDelete
Given API Application with Schema created
And Given create an API Endpoint in the Application
And Given navigate to API Explorer’s created API Application
And GIven I can see the API Schema and API Endpoint
When I delete the API Schema
Then API Schema is not visible in the created API Application in API Explorer

Jira ticket:
https://liferay.atlassian.net/browse/LPS-193293

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are not related to PR
ListAPIApplicationEndpoints#CanFilterEndpointAPIByDateBeforeCreationAsLastUpdated a [test fix ticket](https://liferay.atlassian.net/browse/LRQA-82613) for it
[The others are failed by Unable to update batch engine import task and {code="404", msg="ProxyServlet: /headless-builder/applications", uri=/o/headless-builder/applications}](https://liferay.atlassian.net/browse/LPS-193414)

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+				apiApplicationId = ${staticApplicationId},
+				name = "myEndpoint",
+				path = "/myEndpoint");
+		}
+
+		task ("And Given navigate to API Explorer's created API Application") {
+			APIExplorer.navigateToOpenAPI(customObjectPlural = "my-app");
+		}
+
+		task ("And GIven I can see the API Schema and API Endpoint") {
```
Comment (yanan) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase:111]: I think we may already have related test for this `And GIven I can see the API Schema and API Endpoint`. Could you check and once we confirm it exists then we can remove from here, thanks

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+		}
+	}
+
+	@priority = 4
+	test CanSeeWarningMessageBeforeSchemaIsDeleted {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("When delete API Schema") {
```
Comment (yanan) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase:154]: Would you mind make deleteSchema a macro in APIBuilderUI?

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase (hunk 1 above)
Comment (Hijky) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase:111]: There have been tests that included this assert and if we delete this step, the front step also needs to delete

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**